vanilla_first_setup/defaults/welcome.py

```diff
@@ -57,8 +58,6 @@ class VanillaDefaultWelcome(Adw.Bin):
         'خوش آمدید',
         'आपका स्वागत है',
         'স্বাগতম',
-        'வரவேற்கிறோம்',
```

Comment:
Thanks for your contribution, I will review this PR in detail tomorrow. But removing indic and certain language strings because it moves the UI elements even with a newer implementation isn't a valid solution.